src/lambdas/inspect/audio.js

```diff
@@ -49,6 +54,33 @@ export async function inspect(task, filePath) {
         ...(stream.channel_layout && { Layout: stream.channel_layout }),
       });
     }
+
+    const tags = probe.format?.tags;
+    // Find tags in the format section that match MatchTags from the task
+    if (tags && task.IncludeMetadata) {
+      const keyIncludes = task.IncludeMetadata?.Keys?.StringIncludes;
+      let keysRegex = null;
+      let valuesRegex = null;
+      if (keyIncludes) {
+        keysRegex = new RegExp(keyIncludes);
+      }
+      const valueIncludes = task.IncludeMetadata.Values?.StringIncludes;
+      if (valueIncludes) {
+        valuesRegex = new RegExp(valueIncludes);
+      }
+
+      inspection.Tags = {};
+
+      // use each regex to extract only the matching tags
+      Object.keys(tags).forEach((key) => {
+        const val = tags[key];
+        if (keysRegex?.test(key)) {
```

Comment:
For cases where the API is specifying `StringIncludes`, I would probably use the more simple `includes()` for the test, rather than full-fledged RegExp. I think that behavior would match the naming more closely. And If we don't actually have an immeidate need for RegExps at the moment, I would stick with _only_ `StringIncludes`/`includes()` for now, so we don't have to solve for the additional complexity that comes with well-supported RegExp inputs, like flags and language differences.